app/controllers/licensees_controller.rb

```diff
@@ -1,56 +1,66 @@
 # frozen_string_literal: true
 
 class LicenseesController < ApplicationController
+
   before_action :authenticate_user!
   before_action :authorize_admin, only: %i[new edit]
+    
+    def new
+        @licensee = Licensee.new
+    end
 
-  def new
-    @licensee = Licensee.new
-  end
-
-  def create
-    Licensee.create(
-      facility_name: params[:licensee][:facility_name],
-      contact_name: params[:licensee][:contact_name],
-      address: params[:licensee][:address],
-      city: params[:licensee][:city],
-      state: params[:licensee][:state],
-      zip: params[:licensee][:zip],
-      phone: params[:licensee][:phone],
-      fax: params[:licensee][:fax]
-    )
-    redirect_to licensees_path
-  end
-
-  def index
-    @licensees = Licensee.all
-  end
-
-  def show
-    @licensee = Licensee.find(params[:id])
-  end
-
-  def edit
-    @licensee = Licensee.find(params[:id])
-  end
+    def create
+        Licensee.create(
+            facility_name: params[:licensee][:facility_name],
+            contact_name: params[:licensee][:contact_name],
+            address: params[:licensee][:address],
+            city: params[:licensee][:city],
+            state: params[:licensee][:state],
+            zip: params[:licensee][:zip],
+            phone: params[:licensee][:phone],
+            fax: params[:licensee][:fax],
```

Comment:
Please use strong parameters. 

Using strong parameters is important for form input security and keeping your controller code DRY.
To start, you can read through the short section on strong parameters from the Rails Guides: https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters
Next, to see them in action, read through this section from Hartl's Rails Tutorial book: https://www.railstutorial.org/book/sign_up#sec-strong_parameters